Eevee.Sleep.Bot/Modules/SlashCommands/ChangeRoleSlashModule.cs

```diff
@@ -0,0 +1,64 @@
+using Discord;
+using Discord.Interactions;
+using JetBrains.Annotations;
+using Eevee.Sleep.Bot.Controllers.Mongo;
+using Eevee.Sleep.Bot.Utils;
+
+namespace Eevee.Sleep.Bot.Modules.SlashCommands;
+
+[Group("role", "Commands for role things.")]
+public class ChangeRoleSlashModule : InteractionModuleBase<SocketInteractionContext>
+{
+    [SlashCommand("change", "Select a specific one from the role you have")]
+    [UsedImplicitly]
+    public async Task ChangeRoleAsync()
+    {
+        var roles = UserRoleContoller.FindRoleIdsByUserId(Context.User.Id.ToString());
+        var builder = new ComponentBuilder();
+
+        foreach(var role in roles)
+        {
+            builder.WithButton(
+                label: Context.Guild.Roles.Where(x => x.Id == role).Select(x => x.Name).First(), 
+                customId: $"role{role}"
+            );
+        }
+
+        await Context.Interaction.RespondAsync(
+            "Select your role.",
+            components: builder.Build(),
+            ephemeral: true
+        );
+    }
+
+    [SlashCommand("remove", "Remove role from DB")]
```

Comment:
"Remove the role ownership of a user"

Reply:
Included here
[042bf5a](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25/commits/042bf5ae19def3a76b4fa219af2021bdffdd459e)